source/integrations/zoom.rst

```diff
@@ -1,58 +1,53 @@
-Zoom Plugin 
+Zoom Plugin
 ================================
 
 Mattermost's open source `Zoom plugin <https://github.com/mattermost/mattermost-plugin-zoom>`__ enables users to start and join voice, video and screensharing meetings using either `Zoom <https://zoom.us/>`__'s self-hosted private cloud or on-prem solutions, or its vendor-hosted SaaS solution.
 
-The plugin is installed by default and can be quickly activated. 
+The plugin is installed by default and can be quickly activated.
 
 **Usage**
 
-Once enabled, clicking a video icon in a Mattermost channel invites team members to join a Zoom call, hosted using the credentials of the user who initiated the call. 
+Once enabled, clicking a video icon in a Mattermost channel invites team members to join a Zoom call, hosted using the credentials of the user who initiated the call.
 
 .. image:: https://user-images.githubusercontent.com/177788/42196048-af54d2b8-7e30-11e8-80a0-5e160ae06f03.png
 
 **Using other voice, video and screensharing services**
- 
-The `open source Zoom plugin <https://github.com/mattermost/mattermost-plugin-zoom>`__ can be extended to launch other voice, video and screensharing systems, including custom self-hosted services and vendor-hosted SaaS services such as Skype and Google Hangouts, depending on your needs. 
 
-The following outlines the steps for enabling Zoom. 
+The `open source Zoom plugin <https://github.com/mattermost/mattermost-plugin-zoom>`__ can be extended to launch other voice, video and screensharing systems, including custom self-hosted services and vendor-hosted SaaS services such as Skype and Google Hangouts, depending on your needs.
```

Comment:
Unrelated to your changes, but I don't understand this idea of "extending" the Zoom plugin for other systems. I mean, technically yes they can fork the repository and implement something else using the same button/flow, but this reads like the zoom plugin has built-in support for easy extensibility to other platforms. I'm not sure what to suggest here, since this paragraph is obviously meant to answer, "What if I don't use Zoom?"

Reply:
How about something like this? @lieut-data 

```
The `open source Zoom plugin <https://github.com/mattermost/mattermost-plugin-zoom>`__ can be used as a sample to create other audio and video conferencing extensions, including custom self-hosted services and vendor-hosted SaaS services such as WebEx, Skype for Business and Google Hangouts, depending on your needs.
```

Reply:
Love it, @jasonblais!

Reply:
@svelle Can you help with an update on this PR? Alternatively, I can also submit a separate PR since it's not directly tied to your changes here.

Reply:
Sure thing. Will do in the morning :)

Reply:
Done!

Reply:
Thanks!